api/v1beta1/nodepool_types.go

```diff
@@ -185,6 +185,24 @@ const (
 	UpgradeTypeInPlace = UpgradeType("InPlace")
 )
 
+func (p *UpgradeType) String() string {
+	return string(*p)
+}
+
+func (p *UpgradeType) Set(s string) error {
+	switch s {
+	case string(UpgradeTypeReplace), string(UpgradeTypeInPlace):
+		*p = UpgradeType(s)
+		return nil
+	default:
+		return fmt.Errorf("unknown upgrade type used %s", s)
```

Comment:
nit: Either add a comma after `used` or set `%s` in single quotes to highlight the wrong value used.